config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow]
+# allow FTP traffic after successful HTTP auth
+enabled   = false
+filter    = apache-pass
+banaction = allow-iptables-multiport
+# access log of the website with HTTP auth
+logpath   = /var/log/apache2/access.log
```

Comment:
logpath should be keeping with the other default jail definitions (defined in distro-specific paths). Should be easy to add access, since error logs are already setup this way (assuming access logs aren't already defined).